test/e2e/e2e_test.go

```diff
@@ -43,6 +44,14 @@ import (
 
 var _ = ginkgo.Describe("Kueue visibility server", func() {
 	const defaultFlavor = "default-flavor"
+
+	// We do not check Name and OwnerReference's UID as they are generated
+	// We do not check Controller and BlockOwnerDeletion as we intentionally leave it empty
+	var pendingWorkloadsCmpOpts = []cmp.Option{
+		cmpopts.IgnoreFields(metav1.ObjectMeta{}, "Name"),
```

Comment:
Maybe to clarify we can say `// We do not check workload's Name and its OwnerReference's UID as they are generated at the server-side.`.

> I believe we do check the Namespace in e2e tests.

Yes, we check the namespace of the workload. We do not check the  ownerReference's namespace, because it does not have the `Namespace` field. The owner and the owned object are assumed in the same namespace.